grunnbelop/src/main/kotlin/no/nav/dagpenger/grunnbelop/Grunnbelop.kt

```diff
@@ -50,6 +67,27 @@ fun getGrunnbeløpForDato(dato: LocalDate): Grunnbeløp {
     return grunnbeløp.first { it.gjelderFor(dato) }
 }
 
+fun getGrunnbeløp(): Set<Grunnbeløp> {
+    return grunnbeløp
+}
+
+fun Set<Grunnbeløp>.forDato(dato: LocalDate): Set<Grunnbeløp> {
+    return this.filter { it.gjelderFor(LocalDate.of(dato.year, dato.month, 10)) }.toSet()
```

Comment:
hvorfor den 10ende?

Reply:
🤷‍♂ Det er sånn fra før.

Ikke veldig fornøyd med verken APIet eller hvordan det er å bygge opp strukturen. Det er ganske kaotisk og virker litt sårbart å legge til nytt grunnbeløp når det plutselig er to ulike regler.

Jeg fikler litt på en mer elegant måte, men kommer ikke på noe kjempelurt, så tror jeg heller må skrive gode tester som fanger opp at det mangler mappings.